### DISCUSSION TITLE
Using width attribute for revealjs slides

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#5019] [Q&A] [answered]
I am trying to control the width of a figure in my revealjs slides like this: 

`![](images/linked_in_jobs.png){width=80%}`

But it is not working. When I am in the Quarto document I can see the figure size changing when I change the width attribute, but nothing happens when I render the document or publish it to Quarto Pub. I tried restarting R and shifting to a different machine and am still having the problem. Any tips on how to troubleshoot this issue?

**Comment by cderv** [comment, marked as answer]
You may need to deactivate the auto stretching on image that is applied inside the Revealjs format. 
https://quarto.org/docs/presentations/revealjs/advanced.html#stretch

Did you try that already ?

**Reply by eteitelbaum** [reply]
No but that works, thanks. It sort of messed up the alignment so I just manually set it along with the figure width: 

`![](images/linked_in_jobs.png){fig-align="center" width="60%" }`
